### ISSUE TITLE
Tippy may not be accessible for desktop users using screen-readers

### ISSUE THREAD

**Issue by Drafell** [ryceg/Eigengrau-s-Essential-Establishment-Generator#450]
It looks like there may be some issues with the tool-tip implementation for desktop users that also use a screen-reader or voice assistant. It looks like this might only be binding to the "touchend" or "focus" event and not a "click" or "mouseup" event. This may make the tool-tips effectively invisible for some screen-readers when the browser is being used in a standard desktop mode. Screen-readers will typically look for an actual click-type event in order to read out the dialogue text from the tool-tip.

I am not 100% sure this is an issue as I don't have access to my work tools on this machine and I cannot actually access the generator on that network. Would recommend seeing if we can get actual users of assistive technologies to validate this as an issue.
